src/directory/directory.mjs

```diff
@@ -325,7 +325,12 @@ export const directory = {
           {
             title: 'SMS flows',
             route: '/lib/auth/sms_flows',
-            filters: ['flutter']
+            filters: ['flutter', 'ios']
+          },
+          {
+            title: 'TOTP flows',
```

Comment:
Can we change this to "TOTP two-factor authentication" or something similar?

Reply:
I kept it like this, because SMS was already named like that.. Do we want to rename both?
<img width="326" alt="image" src="https://github.com/aws-amplify/docs/assets/6162866/c78f7356-9b99-4609-b7c2-74ca7b6d2191">